### PR TITLE
Fixes #23276: Improve some messages

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/23276

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-web/doc/dev/desPaquetsDePolitiquesAuxPromisesCFE.mkdwn [hunk 1]
```diff
@@ -1,190 +0,0 @@
-
```
Comment (amousset) [inline webapp/sources/rudder/rudder-web/doc/dev/desPaquetsDePolitiquesAuxPromisesCFE.mkdwn:1]: @fanf this looks very outdated, do you think we need to keep parts of it?

**Comment by amousset** [comment]
PR updated with a new commit

**Review by ElaadF — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**